libs/x-content/src/main/java/org/elasticsearch/common/xcontent/ObjectParser.java

```diff
@@ -164,7 +178,7 @@ public ObjectParser(String name, boolean ignoreUnknownFields, @Nullable Supplier
     }
 
     /**
-     * Creates a new ObjectParser instance with a name.
+     * Creates a new ObjectParser instance with a name that consumes unknown fields as generic Objects.
```

Comment:
Maybe `Creates a new named ObjectParser that consumes unknown fields as generic Objects`?  Otherwise it sounds as though the name is doing the consuming...